templates/zerver/help/configure-who-can-invite-to-streams.md

```diff
@@ -5,6 +5,7 @@
 By default, anyone other than guests can add other users to streams. However, you can restrict this to:
 
 * Organization administrators
+* Organization administrators and moderators
 * Organization administrators and all members
 * Organization administrators and [full members](/help/restrict-permissions-of-new-members)
```

Comment:
We will eventually want to make a /help/ macro for the set of "common policy" options, that we can use in several pages.